docs/UserGuide.md

```diff
@@ -15,17 +16,21 @@ WedLinker is a **desktop app for managing contacts, optimized for use via a  Lin
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have `Java 17` or above installed in your Computer. If you do not have Java 17 installed, or you are unsure, refer to the guide below.
+   1. Download `Java 17` from the official website: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
+   2. Select the installation package based on your Operating System. For Apple users select macOS, for Windows users select Windows.
+   3. Follow the instruction guide to install Java on your device. For more information use this guide: https://docs.oracle.com/en/java/javase/23/install/overview-jdk-installation.html
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest version of WedLinker from [here](https://github.com/AY2425S1-CS2103T-F15-4/tp/releases/). Select `WedLinker.jar` to begin the download.
+![release.png](images/release.png)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your WedLinker.
```

Comment:
Not necessary to change numbering at GHFM/MD helps do the auto numbering and this might make it more difficult for putting a point before this in the future